docs/examples.rst

```diff
@@ -1,31 +1,102 @@
+reproducing relevant results in different areas of machine learning. These
+examples adhere to a shared style and functionality outlined in `#231`_. All
+examples are under the folder `flax/linen_examples/
+<https://github.com/google/flax/tree/master/linen_examples/>`__. Some of the
+examples below have a link [Interactive] that lets you run them directly in
+Colab.
+
+.. _#231: https://github.com/google/flax/issues/231
+
 
 Image classification
 
-   -  `MNIST <https://github.com/google/flax/tree/master/linen_examples/linen_examples/mnist/>`__  [`Interactive <https://colab.research.google.com/github/google/flax/blob/master/linen_examples/mnist/mnist.ipynb>`__]
-   -  `ImageNet <https://github.com/google/flax/tree/master/linen_examples/linen_examples/imagenet/>`__
-   -  `PixelCNN <https://github.com/google/flax/tree/master/linen_examples/linen_examples/pixelcnn/>`__
+   -  `MNIST <https://github.com/google/flax/tree/master/linen_examples/mnist/>`__ [`Interactive
+      <https://colab.research.google.com/github/google/flax/blob/master/linen_examples/mnist/mnist.ipynb>`__] :
+      Convolutional neural network for MNIST classification (featuring simple code).
+   -  `ImageNet <https://github.com/google/flax/tree/master/linen_examples/imagenet/>`__ :
+      Resnet-50 on imagenet with weight decay (featuring multi host SPMD, custom
+      preprocessing, checkpointing, dynamic scaling, mixed precision).
 
 Reinforcement Learning
 
    -  `Proximal Policy
-      Optimization <https://github.com/google/flax/tree/master/linen_examples/linen_examples/ppo/>`__
+      Optimization <https://github.com/google/flax/tree/master/linen_examples/ppo/>`__ :
+      Learning to play Atari games (featuring singel host SPMD, RL setup).
 
 Natural language processing
 
    -  `Sequence to sequence for number
-      addition <https://github.com/google/flax/tree/master/linen_examples/linen_examples/seq2seq/>`__
+      addition <https://github.com/google/flax/tree/master/linen_examples/seq2seq/>`__
+      (featuring simple code, LSTM state handling, on the fly data generation).
    -  `Transformer model on
-      WMT <https://github.com/google/flax/tree/master/linen_examples/linen_examples/wmt/>`__
+      WMT <https://github.com/google/flax/tree/master/linen_examples/wmt/>`__ :
+      Translating English/German (featuring multihost SPMD, dynamic bucketing, attention cache,
+      packed sequences, recipe for TPU training on GCP).
 
 Generative models
 
    -  `Variational
-      auto-encoder <https://github.com/google/flax/tree/master/linen_examples/linen_examples/vae/>`__
\ No newline at end of file
+      auto-encoder <https://github.com/google/flax/tree/master/linen_examples/vae/>`__ :
+      Trained on binarized MNIST (featuring simple code, vmap).
+   -  `PixelCNN++ <https://github.com/google/flax/tree/master/linen_examples/pixelcnn/>`__ :
+      Trained on cifar10 (featuring single host SPMD, checkpointing, Polyak decay).
+
+
+Flax examples from the community
```

Comment:
Moved into a separate section.

@danieljtait
@henry-prior
@joaogui1 
@PForet
@bkkaggle 

It would be awesome if you could update your examples to the new API - Avital can point you to the update guide that makes this process pretty straightforward...

Reply:
I updated my example locally a few weeks ago but ill clean it up and push it up hopefully today

Reply:
Hi folks! More details on Linen can be found here: https://github.com/google/flax/blob/master/flax/linen/README.md

And here is the [upgrade guide](https://docs.google.com/document/d/1hYavTVPaKVVe9Be8pCB7yW7r6dDv3RALVNit8NZca4c).

If you have questions about Linen please do reach out on GitHub discussions -- we try to respond to those regularly.

Reply:
I've updated https://github.com/henry-prior/jax-rl to Linen, thanks for the heads-up!

Reply:
@bkkaggle and @henry-prior, can you please file pull requests modifying this README to put your examples in the right place?